src/Pages/Signup/Signup.js

```diff
@@ -190,10 +189,11 @@ class Signup extends Component {
       password === pwCheck &&
       password === repassword
     ) {
-      fetch('http://10.58.2.7:8000/users/signup', signup_info)
+      fetch('http://10.58.6.178:8000/users/signup', signup_info)
```

Comment:
```suggestion
      fetch(`${API_URL}/users/signup`, signup_info)
```
만약 api 주소가 바뀐다면 쓰이는 곳마다 찾아다니면서 주소를 바꿔주어야 합니다. src/config.js 등의 장소에 상수 변수로 저장해두고 관리하면 훨씬 편할 것 같네요!